src/public/js/ArchivalSpace/index.js

```diff
@@ -1,7 +1,8 @@
+import {systemConfirmMsg_archivalRoom} from '@js/constants';
```

Comment:
for readability, we would want to keep camelcase and snake case consistent. So `systemConfirmMsgArchivalRoom`?

Reply:
Sounds good, resolved in this [commit](https://github.com/together-support/togethernet/pull/57/commits/60d1fedd0a04ac38cd46112a5d70fd55dbbe0cee).